Header header
string description
uint8 extrude
string altitudeMode
coordinates coord
